packages/product-react/stories/List.story.tsx

```diff
@@ -0,0 +1,111 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { List, IListProps, IItems } from '../src/components/List/List';
+import { Checkbox } from '../src/components/Checkbox';
+import '../src/components/List/List.scss';
+
+const customListItem: React.FC<IListProps> = ({
+  items,
+  className,
+  ...rest
```

Comment:
looks like this can be removed

Reply:
wait this is a story so it doesn't matter too much

Reply:
yes it is, it's just to demo a custom component